Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,412 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<estw>
+  <station name="Radolfzell" short="RRZ" refNr="56">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="66BP" direction="right" />
+      <block name="56BN2" direction="right" />
+    </track>
+  </station>
+
+  <station name="Schwackenreute" short="RSU" refNr="65">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="65BN" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Eriskirch" short="TEK" refNr="76">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="77BP2" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Baienfurt" short="TBAG" refNr="40">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="40BN2" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Aulendorf" short="TAU" refNr="38">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="38BP69" direction="left"/>
+      <block name="39BP" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Brandbühl" short="RBRB" refNr="66" scheduleFile="g66_____.abf" localOrderFile="bf66____.abf">
+    <track name="52" isPlatform="false">
+      <block name="66G52"/>
```

Comment:
Dieses ZN-Feld gibt es, soweit ich das überblicke, nicht. Hier sollten vier Blöcke angegeben sein: P und N2 in Fahrtrichtung links sowie P und N in Fahrtrichtung rechts. Dann ist sichergestellt, dass Brandbühl erst als aktuelle Betriebsstelle angezeigt wird, wenn diese tatsächlich passiert wurde.

Reply:
ok, verstanden wie es tickt. Ist angepasst.